### ISSUE TITLE
Divergenge in no-log behavior recommendations for Apache2 and Nginx

### ISSUE THREAD

**Issue by heartsucker** [freedomofpress/securedrop#4255]
## Description

We have the following block in our docs that explains how to remove logging for the SecureDrop landing page on an org's website.

```rst
**Don't log access to the *Landing Page* in the webserver**

Here's an Apache example that would exclude the *Landing Page* from
logging. However you still need to make sure no other assets get logged!

::

    SetEnvIf Request_URI "^/securedrop($|(\/.*))" dontlog
    CustomLog logs/access_log common env=!dontlog

In nginx, logging can be disabled like so:

::

    access_log off;
    error_log /dev/null;
```

In this example, the Apache directive has a conditional `dontlog` based off the presence of the URI. The Nginx example does not. We should revise the Nginx example to show how and admin may invoke the same behavior on Nginx.

This was noticed in reviewing #4238.

**Comment by rmol** [comment]
Is there a reason this wasn't closed after #4370 was merged?

**Comment by redshiftzero** [comment]
good point @rmol!

closed by #4370

**Comment by redshiftzero** [comment]
(and the reason was the GitHub auto close keywords won't work for issue B in "Fixes #A and #B"
